### DISCUSSION TITLE
8BitDo n64 Mod Kit

### DISCUSSION THREAD

**Discussion by lamprey13** [darthcloud/BlueRetro#743] [General]
I've seen a couple of reports of issues with the 8BitDo n64 Mod Kit and the Blueretro receiver. Unfortunately, I'm having issues with the button mappings. I'm using the 1.8.3 release of the blueretro firmware.

1. Is this a known issue?
2. Is it something that I can "fix" by changing the mappings? If so, is there known good configuration/setting? I tried adjusting the mappings, but I couldn't get it to work right.

I wasn't sure if this is something I can change settings for to get working or if it requires a firmware update on the blueretro side of things. 

Happy to provide more information if it helps.

Cheers!

**Comment by darthcloud** [comment]
I ordered one, and when I receive it, I'll fix whatever isn't right by default.

Meanwhile it's possible changing mapping might help. But I don't know.

**Comment by lamprey13** [comment]
Thanks. Appreciate the update and your hard work. I'll keep a look out for an update!

**Reply by mistermorcus** [reply]
There is some good info in this issue https://github.com/darthcloud/BlueRetro/issues/706. 

Someone listed out the mapping changes necessary to get the controller working when connecting after the first time pairing.  If you want to re-pair the controller after every use than the mapping is always correct. If you just want to turn it on and use it, you need to do the button mappings. However, C-Right does not work as noted in the thread. Rumble also does not work in any configuration.

**Reply by lamprey13** [reply]
Thanks. 

 I did see that thread and gave mapping a go. But, since it's not 100% (c-right) I'll just wait until the firmware is updated. Fortunately, I have a couple of Brawler 64 wireless controllers I can use, but just looking for that authentic feel without wires. :)

**Comment by lamprey13** [comment]
Just wanted to say thanks for getting this sorted out and working. Seems to be working great for me.

I didn't see in the release notes anything about runble. Is that not supported, yet? Or do I need to do something on my end enable rumble support?

Thanks!

**Reply by darthcloud** [reply]
If you click on the docs link that is in the release note the info is there 😛

**Reply by lamprey13** [reply]
Doh. Skimmed right past it. Thanks! :)
